### PR TITLE
Raise to 9.4.0

### PR DESCRIPTION
**Pull request by jackra1n**

### CODE REVIEW

File: alfresco-connector/pom.xml [hunk 1]
```diff
@@ -1,4 +1,6 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (ivy-rew) [inline alfresco-connector/pom.xml:2]: where does this re-formatting com from? If it's your editor I'd avoid that in sub-sequent PRs  ... very likely a simple save operation with our 'deployment editor' in the designer will format this file differently.

File: alfresco-connector-test/pom.xml [hunk 2]
```diff
@@ -52,5 +45,13 @@
         </plugin>
       </plugins>
     </pluginManagement>
+    <plugins>
```
Comment (ivy-rew) [inline alfresco-connector-test/pom.xml:48]: do not move me please ;-) ... next time!

**Review by ivy-rew — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -16,7 +16,7 @@ jobs:
     - name: Setup Java JDK
       uses: actions/setup-java@v1.4.3
       with:
-        java-version: 11
+        java-version: 17
```
Comment (alexsuter) [inline .github/workflows/ci.yml:19]: @jackra1n  any reasons why you didn't change to our reusable pipelines?

Comment (jackra1n) [reply .github/workflows/ci.yml:19]: I didnt think about it 🤔. I dont think we talked about those and I didn't want to break anything 😬 
If I finish my sprint tasks quickly and there wont be anything else important I can help with, I will work on switching those to our reuseable pipelines

Comment (alexsuter) [reply .github/workflows/ci.yml:19]: I'll do it.

**Pull request merged**